src/PhpGitHooks/Application/PhpUnit/PhpUnitHandler.php

```diff
@@ -46,7 +46,7 @@ public function run()
         }
     }
 
-    private function setTitle()
+    protected function setTitle()
```

Comment:
why is this protected now? you override the function anyway in the `PhpUnitCheckCodeCoverageHandler` class